src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx

```diff
@@ -0,0 +1,550 @@
+    string interactionId = CrossNewRelic.Current.StartInteraction("Getting data from service");
+
+    var response = await myClient.GetAsync(new Uri("https://jsonplaceholder.typicode.com/todos/1"));
+    if (response.IsSuccessStatusCode)
+    {
+        var content = await response.Content.ReadAsStringAsync();
+    } else
+    {
+        Console.WriteLine("Unsuccessful response code");
+    }
+
+    CrossNewRelic.Current.EndInteraction(interactionId);
+
+```
+
+</Collapser>
+
+<Collapser
+    id="record-transactions"
+    title="Record HTTP transactions"
+>
+    **[NoticeHttpTransaction](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/notice-http-transaction/)(string url, string httpMethod, int statusCode, long startTime,long endTime, long bytesSent, long bytesReceived, string responseBody): void;**
+
+Tracks network requests manually. You can use this method to record HTTP transactions, with an option to also send a response body.
+
+``` C#
+    CrossNewRelic.Current.NoticeHttpTransaction(
+      "https://newrelic.com",
+      "GET",
+      200,
+      DateTimeOffset.Now.ToUnixTimeMilliseconds(),
+      DateTimeOffset.Now.ToUnixTimeMilliseconds() + 100,
+      0,
+      1000,
+      ""
+    );
+```
+</Collapser>
+
+<Collapser
+    id="record-failures"
+    title="Record network failures"
+>
+    **[NoticeNetworkFailure](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/notice-network-failure/)(string url, string httpMethod, int statusCode, long startTime,long endTime, long bytesSent, long bytesReceived, string responseBody): void;**
+
+Records network failures. If a network request fails, use this method to record details about the failure.
+
+``` C#
+    CrossNewRelic.Current.NoticeNetworkFailure(
+      "https://fakewebsite.com",
+      "GET",
+      DateTimeOffset.Now.ToUnixTimeMilliseconds(),
+      DateTimeOffset.Now.ToUnixTimeMilliseconds() + 100,
+      NetworkFailure.Unknown
+    );
+```
+</Collapser>
+
+<Collapser
+    id="breadcrumbs"
+    title="Record breadcrumbs"
+>
+    **[RecordBreadcrumb](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/recordbreadcrumb)(string name, Dictionary{'<string, object>'} attributes): bool;**
+
+This call creates and records a `MobileBreadcrumb` event, which can be queried with NRQL and viewed in a crash event trail.
+
+``` C#
+    CrossNewRelic.Current.RecordBreadcrumb("MAUIExampleBreadcrumb", new Dictionary{'<string, object>'}()
+        {
+            {"BreadNumValue", 12.3 },
+            {"BreadStrValue", "MAUIBread" },
+            {"BreadBoolValue", true }
+        }
+    );
+```
+</Collapser>
+
+<Collapser
+    id="custom-events"
+    title="Record custom events"
+>
+    **[RecordCustomEvent](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/recordcustomevent-android-sdk-api)(string eventType, string eventName, Dictionary{'<string, object>'} attributes): bool;**
+
+Creates and records a custom event for use in New Relic Insights.
```

Comment:
@ndesai-newrelic I tried searching for New Relic Insights but couldn't find a good doc to link to.